spark/src/main/scala/geotrellis/spark/tiling/CutTiles.scala

```diff
@@ -16,19 +16,20 @@
 
 package geotrellis.spark.tiling
 
-import geotrellis.vector.Extent
 import geotrellis.raster._
 import geotrellis.raster.merge._
 import geotrellis.raster.prototype._
 import geotrellis.raster.resample._
 import geotrellis.spark._
+import geotrellis.util.LazyLogging
 
-import org.apache.spark._
 import org.apache.spark.rdd._
 
 import scala.reflect.ClassTag
 
 object CutTiles {
+  @transient private lazy val logger = LazyLogging(this)
```

Comment:
Yeah, that was the intent is to avoid adding types to classes when all we want is this private field. I think this should be the prefered approach.